gateway/1.4/modules/ROOT/pages/policies-automated-overview.adoc

```diff
@@ -0,0 +1,2 @@
+:page-aliases: api-manager::automated-policies-landing-page.adoc
+include::policies::policies-automated-overview.adoc[]
```

Comment:
Just a q - does this include work right after the page alias?

Reply:
Seems to work. I've built locally and the pages render correctly.